lib/Core/Pagerfanta.php

```diff
@@ -546,4 +556,14 @@ private function needsToIntegerConversion($value): bool
     {
         return (\is_string($value) || \is_float($value)) && (int) $value == $value;
     }
+
+    /**
+     * @param int $maxNbPages
+     */
+    public function setMaxNbPages($maxNbPages): self
```

Comment:
```suggestion
    public function setMaxNbPages(int $maxNbPages): self
```

It's a new method, we can add the typehint now.

Reply:
Totally agree.